Sources/_FoundationInternals/String/String+Internals.swift

```diff
@@ -16,4 +16,32 @@ extension String {
             $0.properties.isWhitespace
         })
     }
+
+    static func _tryFromUTF16(_ input: UnsafeBufferPointer<UInt16>) -> String? {
+        // Allocate input.count * 3 code points since one UTF16 code point may require up to three UTF8 code points when transcoded
+        withUnsafeTemporaryAllocation(of: UInt8.self, capacity: input.count * 3) { contents in
```

Comment:
```suggestion
        withUnsafeTemporaryAllocation(of: UTF8.CodeUnit.self, capacity: input.count * 3) { contents in
```
Extremely minor nit.